rpm/tayga.spec

```diff
@@ -0,0 +1,60 @@
+Name:		tayga
+Version:	0.9.2
+Release:	1
+Summary:	Simple, no-fuss NAT64
+License:	GPLv2+
+URL:		http://www.litech.org/%{name}/
+Source0:	http://www.litech.org/%{name}/%{name}-%{version}.tar.bz2
```

Comment:
You can remove the remote address here we don't use the tarball.

Reply:
In many of our packages we do include that source and URL. Which could be returned back to spec?

I guess it is more important to keep the `URL`